tests/_utils/model.js

```diff
@@ -3,20 +3,41 @@
  * For licensing, see LICENSE.md.
  */
 
-import TreeWalker from '/ckeditor5/engine/model/treewalker.js';
+import Mapper from '/ckeditor5/engine/conversion/mapper.js';
+import Document from '/ckeditor5/engine/model/document.js';
```

Comment:
You could use `ModelDocument`, `ModelRange` and `ModelPosition` not to confuse with native one, especially if you have `ModelElement` below.